hack/build-coreos-manifest.go

```diff
@@ -20,14 +20,20 @@ import (
 const (
 	streamRHCOSJSON = "data/data/coreos/rhcos.json"
 	streamFCOSJSON  = "data/data/coreos/fcos.json"
+	streamSCOSJSON  = "data/data/coreos/scos.json"
 	fcosTAG         = "okd"
+	scosTAG         = "scos"
 	dest            = "bin/manifests/coreos-bootimages.yaml"
 )
 
 func run() error {
 	streamJSON := streamRHCOSJSON
-	if tags, _ := os.LookupEnv("TAGS"); strings.Contains(tags, fcosTAG) {
-		streamJSON = streamFCOSJSON
+	if tags, _ := os.LookupEnv("TAGS") {
+		if strings.Contains(tags, fcosTAG) {
+			streamJSON = streamFCOSJSON
+		} else if  strings.Contains(tags, scosTAG) {
+			streamJSON = streamSCOSJSON
+		}
```

Comment:
nit: seems like switch time

```suggestion
var streamJSON string
switch {
  case strings.Contains(tags, fcosTAG):
    streamJSON = streamFCOSJSON
   strings.Contains(tags, scosTAG):
    streamJSON = streamSCOSJSON
   default:
    streamJSON = streamRHCOSJSON
}
```